app/assets/javascripts/authoring/events.js

```diff
@@ -362,12 +362,12 @@ function showDropDown(){
         },
         items: {
             "view": {name: "View", icon: ""},
-            "edit": {name: "Edit", icon: "", disabled: function(key, opt) { 
+            "edit": {name: "Edit", icon: "", disabled: function(key, opt) {
```

Comment:
Seems like there are a lot of these lines where only whitespace was changed. Could you revert them back to the original whitespace, so that this PR only touches the lines it directly changes?